xapian-applications/omega/docs/overview.rst

```diff
@@ -315,7 +315,8 @@ other filters too - see below):
 * TCR (simple compressed text format) files (.tcr) if libe-book is available
 * eReader files (.pdb) if libe-book is available
 * Sony eBook files (.lrf) if libe-book is available
-* Image files that contain text (.png, .jpg) if libtesseract-dev is available.
+* Image JPEG files that contain text (.png, .jpg, .jpeg, .jfif, .jpe, .webq, .tif, .tiff,
+  .pbm, .gif, .ppm, .pgm) if libtesseract-dev is available.
```

Comment:
The change from "Image files" to "Image JPEG files" doesn't seem helpful.

Also, `.webq` rather than `.webp` here - is that a typo?

Reply:
Yes, it is a mistake.